Sync fails on unitymedia DS-Lite network
User Agent: Mozilla/5.0 (X11; Linux i686; rv:34.0) Gecko/20100101 Firefox/34.0
Build ID: 20141201180357

Steps to reproduce:

* I am running Arch Linux with 32 bit, Firefox Version 34.0.5.
* I ran rm -rf ~/.mozilla/ to remove all user config files.
* I changed logonsuccess in about:config to true to see all log
* After that I created a new sync account, and under manage I excluded everything from beeing synced except booksmarks.
* Only this client is connected to this account.
* Then I triggered manually a sync


Actual results:

1420667811202 Sync.ErrorHandler DEBUG Flushing file log.
1420667811205 Sync.Status DEBUG Status.service: error.sync.failed_partial => success.status_ok
1420667811208 Sync.ErrorHandler DEBUG Log cleanup threshold time: 1419803811208
1420667811208 Sync.ErrorHandler DEBUG No log to clean up.
1420667812194 Sync.Tracker.Bookmarks DEBUG Saving changed IDs to bookmark
1420668057692 Sync.Status DEBUG Status.service: success.status_ok => success.status_ok
1420668057692 Sync.Status DEBUG Status.service: success.status_ok => success.status_ok
1420668057695 Sync.ErrorHandler DEBUG Beginning user-triggered sync.
1420668057695 Sync.Service DEBUG User-Agent: Firefox/34.0.5 FxSync/1.36.0.20141201180357.
1420668057696 Sync.Service INFO Starting sync at 2015-01-07 23:00:57
1420668057696 Sync.SyncScheduler DEBUG Clearing sync trigger and the global score.
1420668057699 Sync.Status INFO Resetting Status.
1420668057699 Sync.Status DEBUG Status.service: success.status_ok => success.status_ok
1420668057701 Sync.BrowserIDManager DEBUG _ensureValidToken already ha one
1420668059155 Sync.Resource DEBUG mesg: GET success 200 https://sync-189-us-west-2.sync.services.mozilla.com/1.5/17661343/info/collections
1420668059155 Sync.Resource DEBUG GET success 200 https://sync-189-us-west-2.sync.services.mozilla.com/1.5/17661343/info/collections
1420668059158 Sync.Service DEBUG Fetching global metadata record
1420668059159 Sync.Service DEBUG Weave Version: 1.36.0 Local Storage: 5 Remote Storage: 5
1420668059159 Sync.Service INFO Sync key is up-to-date: no need to upgrade.
1420668059159 Sync.Service DEBUG Fetching and verifying -- or generating -- symmetric keys.
1420668059159 Sync.Service INFO Testing info/collections: {"meta":1420667325.98,"crypto":1420666971.7,"clients":1420667685.57}
1420668059159 Sync.CollectionKeyManager INFO Testing for updateNeeded. Last modified: 1420666971.7
1420668059159 Sync.Synchronizer DEBUG Refreshing client list.
1420668059161 Sync.Engine.Clients INFO 0 outgoing item pre-reconciliation
1420668059162 Sync.Engine.Clients INFO Records: 0 applied, 0 successfully, 0 failed to apply, 0 newly failed to apply, 0 reconciled.
1420668059163 Sync.Synchronizer INFO Updating enabled engines: 2 clients.
1420668059168 Sync.Engine.Bookmarks DEBUG First sync, uploading all item
1420668059178 Sync.Store.Bookmarks DEBUG No node for GUID mobile; returning no children.
1420668059178 Sync.Engine.Bookmarks INFO 12 outgoing item pre-reconciliation
1420668059179 Sync.Engine.Bookmarks DEBUG Bookmarks backup starting.
1420668059197 Sync.Engine.Bookmarks DEBUG Bookmarks backup done.
1420668059198 Sync.Engine.Bookmarks INFO Records: 0 applied, 0 successfully, 0 failed to apply, 0 newly failed to apply, 0 reconciled.
1420668059205 Sync.Store.Bookmarks DEBUG No node for GUID mobile; returning no children.
1420668059242 Sync.Engine.Bookmarks INFO Uploading all of 12 record
1420668059243 Sync.BrowserIDManager DEBUG _ensureValidToken already ha one
1420668059245 Sync.Collection DEBUG POST Length: 6066
1420668060161 Sync.Tracker.Clients DEBUG Saving changed IDs to client
1420668060178 Sync.Tracker.Bookmarks DEBUG Saving changed IDs to bookmark
1420668184605 Sync.Status DEBUG Status.sync: success.sync => error.login.reason.network
1420668184605 Sync.Status DEBUG Status.service: success.status_ok => error.sync.failed
1420668184606 Sync.Status DEBUG Status for engine bookmarks: error.engine.reason.unknown_fail
1420668184606 Sync.Status DEBUG Status.service: error.sync.failed => error.sync.failed_partial
1420668184606 Sync.ErrorHandler DEBUG bookmark failed: Error: NS_ERROR_NET_RESET (resource://services-sync/resource.js:394:18) JS Stack trace: Res_post@resource.js:428:12 < SyncEngine.prototype._uploadOutgoing/doUpload<@engines.js:1384:20 < innerBind@util.js:525:42 < SyncEngine.prototype._uploadOutgoing@engines.js:1432:9 < SyncEngine.prototype._sync@engines.js:1483:7 < wrappedSync@bookmarks.js:214:11 < _sync@bookmarks.js:211:1 < WrappedNotify@util.js:148:21 < Engine.prototype.sync@engines.js:655:5 < _syncEngine@enginesync.js:199:7 < sync@enginesync.js:149:15 < onNotify@service.js:1275:7 < WrappedNotify@util.js:148:21 < WrappedLock@util.js:103:16 < _lockedSync@service.js:1265:12 < sync/<@service.js:1257:14 < WrappedCatch@util.js:77:16 < sync@service.js:1245:5
1420668184607 Sync.Synchronizer INFO Sync completed at 2015-01-07 23:03:04 after 126.91 secs.
1420668184608 Sync.Declined DEBUG Handling remote declined: []
1420668184609 Sync.Declined DEBUG Handling local declined: []
1420668184609 Sync.Declined DEBUG Declined changed? false
1420668184609 Sync.Service INFO No change to declined engines. Not reuploading meta/global.
1420668184610 Sync.SyncScheduler DEBUG Next sync in 600000 ms.
1420668184611 Sync.ErrorHandler DEBUG Some engine did not sync correctly.



Expected results:

I am having this bug since a few month, I think it started someday in November. I thought it something temporarely, but it not.
